core/src/i18n/en.yaml

```
@@ -716,12 +716,25 @@ functions:
       text: Warning
       variable-can-override-sbu: This variable can override or be overridden by a variable injected by one of the created ServiceBindings.
 gateways:
+  name_singular: Gateway
+  description: Gateway describes a load balancer operating at the edge of the mesh receiving incoming or outgoing HTTP/TCP connections.
   create-modal:
+    at-least-one-server-required: Gateway must have at least one server.
+    tooltips:
+      ca-certificates: The path to a file containing certificate authority certificates to use in verifying a presented client side certificate.
```

```suggestion
      ca-certificates: The path to a file containing certificate authority (CA) certificates, which verify a presented client-side certificate.
```

I'm adding the `(CA)` just to break up the 3 words, I had to read this 3 times till I was sure it's not just a duplication of words :D